gems/pending/openstack/openstack_event_monitor.rb

```diff
@@ -102,6 +99,14 @@ def self.event_monitor_key(options)
     options.values_at(:hostname, :username, :password)
   end
   private_class_method :event_monitor_key
+
+  def amqp_event(_delivery_info, metadata, payload)
+    OpenstackAmqpEvent.new(payload,
```

Comment:
Can you rename this class to just `OpenstackEvent`?  Since now it's used in more than just `amqp` event monitors.

Then, this method should be renamed to `openstack_event`.  Or, just update the `OpenstackEvent#initialize` method.